ecosystem/resources/members/qiskit-on-iqm.toml

```diff
@@ -0,0 +1,8 @@
+name = "Qiskit on IQM"
+url = "https://github.com/iqm-finland/qiskit-on-iqm"
+description = "Qiskit adapter for IQM's quantum computers."
+licence = "Apache 2.0"
+labels = []
+created_at = 1715085233.375706
```

Comment:
Nit: seems unnecessarily precise to have decimals here. Not a big deal though

Reply:
Haha yeah. They're all like that though; it's just the output of `datetime.now()`. I'm gonna leave it for consistency.